app/views/faculty_surveys/_listing.html.erb

```diff
@@ -3,7 +3,7 @@
     <table class="mi_table">
       <thead class="mi_thead">
         <tr>
-          <th class="header_th">  </th>
+          <td></td>
```

Comment:
What is the purpose of this empty tag ?

Reply:
To create a column where the Edit links will be displayed
![Screenshot 2023-05-22 at 3 58 56 PM](https://github.com/lsa-mis/lsa_ride_share/assets/56089936/42add08c-4b60-469e-bd0a-22512360ca94)